src/middlewared/middlewared/api/v25_10_0/acl.py

```diff
@@ -108,15 +134,37 @@ def check_inherit_only(self) -> Self:
 
 class NFS4ACE_BasicFlags(BaseModel):
     BASIC: NFS4ACE_BasicFlagset
+    """Basic inheritance behavior for NFS4 ACE.
+
+    * `INHERIT`: Apply to child files and directories
+    * `NOINHERIT`: Do not apply to child objects
+    """
 
 
 class NFS4ACE(BaseModel):
     tag: NFS4ACE_Tags
+    """Subject type for this ACE.
+
+    * `SPECIAL_OWNER`: File/directory owner
+    * `SPECIAL_GROUP`: File/directory primary group
+    * `SPECIAL_EVERYONE`: All users
```

Comment:
These summaries are incorrect. The string field is `owner@`, `group@`, and `everyone@`.